### PR TITLE
Adding additional metrics in placement service

### PR DESCRIPTION
**Pull request by shalabhms**
# Description
Adding a few additional metrics in Placement Service using OpenCensus. 

	"placement/hosts_total" :  "The total number of active hosts reported to placement service.",
	"placement/actortypes_total" : "The total number of active actor types reported to placement service.",
	"placement/nonactorhosts_total" : "The total number of active non actor hosts reported to placement service.",

## Issue reference

#971

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation
* [ ] Specification has been updated
* [ ] Provided sample for the feature

### CODE REVIEW

File: pkg/placement/monitoring/metrics.go [hunk 1]
```diff
@@ -18,13 +18,31 @@ var (
 		"placement/hosts_total",
 		"The total number of active hosts reported to placement service.",
 		stats.UnitDimensionless)
+	activeActorTypesTotal = stats.Int64(
+		"placement/actorTypes_total",
```
Comment (youngbupark) [inline pkg/placement/monitoring/metrics.go]: Can we use lowercased metric name ?
`placement/actor_types_total`

File: pkg/placement/monitoring/metrics.go [hunk 2]
```diff
@@ -18,13 +18,31 @@ var (
 		"placement/hosts_total",
 		"The total number of active hosts reported to placement service.",
 		stats.UnitDimensionless)
+	activeActorTypesTotal = stats.Int64(
+		"placement/actorTypes_total",
+		"The total number of active actor types reported to placement service.",
+		stats.UnitDimensionless)
+	activeNonActorTypesTotal = stats.Int64(
+		"placement/nonactorTypes_total",
```
Comment (youngbupark) [inline pkg/placement/monitoring/metrics.go]: same here.

File: pkg/placement/monitoring/metrics.go (hunk 1 above)
Comment (shalabhms) [reply pkg/placement/monitoring/metrics.go]: done.

File: pkg/placement/monitoring/metrics.go (hunk 2 above)
Comment (shalabhms) [reply pkg/placement/monitoring/metrics.go]: changed to lowercase.

File: pkg/placement/monitoring/metrics.go [hunk 3]
```diff
@@ -18,13 +18,31 @@ var (
 		"placement/hosts_total",
 		"The total number of active hosts reported to placement service.",
 		stats.UnitDimensionless)
+	activeActorTypesTotal = stats.Int64(
+		"placement/actortypes_total",
+		"The total number of active actor types reported to placement service.",
```
Comment (amanbha) [inline pkg/placement/monitoring/metrics.go]: active can be removed from here.

File: pkg/placement/monitoring/metrics.go [hunk 4]
```diff
@@ -18,13 +18,31 @@ var (
 		"placement/hosts_total",
 		"The total number of active hosts reported to placement service.",
 		stats.UnitDimensionless)
+	activeActorTypesTotal = stats.Int64(
+		"placement/actortypes_total",
+		"The total number of active actor types reported to placement service.",
+		stats.UnitDimensionless)
+	activeNonActorTypesTotal = stats.Int64(
+		"placement/nonactortypes_total",
+		"The total number of active non actor types reported to placement service.",
```
Comment (amanbha) [inline pkg/placement/monitoring/metrics.go]: This should changed to The total number of non-actor hosts reported to placement service.

File: pkg/placement/monitoring/metrics.go [hunk 5]
```diff
@@ -18,13 +18,33 @@ var (
...
 	stats.Record(context.Background(), activeHostsTotal.M(int64(count)))
 }
 
+// RecordActiveActorTypesCount records the number of active actor types
+func RecordActiveActorTypesCount(count int) {
+	stats.Record(context.Background(), actorTypesTotal.M(int64(count)))
+}
+
+// RecordActiveNonActorHostsCount records the number of active non actor hosts
+func RecordActiveNonActorHostsCount(count int) {
```
Comment (youngbupark) [inline pkg/placement/monitoring/metrics.go]: We may need to rename this method  name without `Active` as well.

Comment (shalabhms) [reply pkg/placement/monitoring/metrics.go]: fixed now

**Pull request merged**